advisories/github-reviewed/2022/05/GHSA-65j5-vpm7-6xp4/GHSA-65j5-vpm7-6xp4.json

```diff
@@ -32,10 +32,7 @@
             }
           ]
         }
-      ],
-      "database_specific": {
-        "last_known_affected_version_range": "< 3.1.33-dev-4"
-      }
+      ]
```

Comment:
Note: I provided the patch via the UI, so I don't know if removal of the `database_specific` block is actually correct.